### ISSUE TITLE
i like this great project

### ISSUE THREAD

**Issue by sir-reynolds** [Moxio/sqlite-extended-api#2]
dear sir.

my question is, can i use this wrapper to connect sqlite without pdo?

example : new SQlite3( 'db' );

**Comment by aboks** [comment]
Thanks for your interest in this project! Currently, this projects only wraps PDO connections through `Facade::wrapPDO()`, but it is probably possible to also offer a `Facade::wrapSQLite3()` to wrap connections made through PHP's SQLite3 extension. Please note that loading SQLite extensions (which was one of the main purposes for this library, and is not possible with plain PDO) is already possible when using the SQLite3 extension, through https://www.php.net/manual/en/sqlite3.loadextension.php. Is there any specific function in the underlying SQLite C library that you want to use through the SQLite3 extension?

**Comment by sir-reynolds** [comment]
hello,

yes i know about php sqlite3 loadextension, but when i try load module .so that always fail . and then i search info most said its because sqlite3 C enable extension is false.

i use php7.2 and sqlite3 just version 3.11 . i need sqlite3 version >= 28 with fts5 enable. i try use php 7.4 it's same have sqlite3 version 3.11.

i very need sqlite3 fts5

**Comment by aboks** [comment]
I just released version 0.3.0, which supports wrapping connections made through PHP's SQLite3 extension:
```php
$db = new SQLite3("path/to/db");
$wrapped = Facade::wrapSQLite3($db);
```

Just be aware that this cannot do anything the underlying SQLite C library/binary cannot do. If you want to test whether SQLite itself can load the extension, you could open a shell on your server (e.g. through SSH), run the `sqlite3` command, and attempt to load your extension using something like
```
sqlite> .load mod_someextension.so
```
If that does not work, the issue cannot be solved by PHP and you should first aim to create a setup that can successfully execute the steps above. 

Also note that as of PHP 7.4, the libsqlite3 library is no longer bundled with PHP. If you want to upgrade your SQLite3 version, you should find an appropriate package for your OS.
